### PR TITLE
made easier for etherpad discussion

### PR DESCRIPTION
**Pull request by raynamharris**
added 1), 2), and 3) so that people can respond with 1 2 3 in the etherpad

### CODE REVIEW

**Comment by lexnederbragt** [comment]
Good! But maybe you need to add some newlines so that these become a numbered list (unless you specifically don't want that).

**Comment by raynamharris** [comment]
Good point @lexnederbragt. How's the change I just made?

**Comment by lexnederbragt** [comment]
Looks OK in markdown, but not yet in the rendered one. I think you need an empty line before the first number...

File: _episodes/02-novice.md [hunk 1]
```diff
@@ -294,9 +294,11 @@ what to explain to that particular learner next.
 
 > ## Handling Outcomes
 >
-> As the instructor, what should you do if most of the class votes for one
-> of the wrong answers?  For the right answer?  What if the votes are evenly
-> spread between options?
+> What should the instructor do when, the class votes for: 
+>
+> 1. mostly the wrong answers?
```
Comment (ChristinaLK) [inline _episodes/02-novice.md]: This should be, specifically: 
1. one of the wrong answers

File: _episodes/02-novice.md [hunk 2]
```diff
@@ -294,9 +294,11 @@ what to explain to that particular learner next.
 
 > ## Handling Outcomes
 >
-> As the instructor, what should you do if most of the class votes for one
-> of the wrong answers?  For the right answer?  What if the votes are evenly
-> spread between options?
+> What should the instructor do when, the class votes for:
```
Comment (ChristinaLK) [inline _episodes/02-novice.md]: while I'm suggesting another change, the comma isn't needed

**Comment by ErinBecker** [comment]
Merged! Thanks @raynamharris, @lexnederbragt  and @ChristinaLK.

**Pull request merged**